client/src/pages/OnBoardingPage.tsx

```diff
@@ -1,9 +1,120 @@
-import React from 'react'
+import React, { useRef, useState } from "react";
+import styles from "./OnBoardingPage.module.css";
+import { IconLeetCodeLogo } from "../icons/icon-leetcode-logo";
+import { IconVJudgeLogo } from "../icons/icon-vjudge-logo";
+import { IconAddPhoto } from "../icons/import-photo-icon";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
 
 const OnBoardingPage = () => {
+  const hiddenFileInput = useRef<HTMLInputElement>(null);
+  const [imageUrl, setImageUrl] = useState<string | null>(null);
+  const [leetCodeUsername, setLeetCodeUsername] = useState("");
+  const [vJudgeUsername, setVJudgeUsername] = useState("");
+  const [errorMsg, setErrorMsg] = useState("");
+  const navigate = useNavigate();
+
+  const handleClick = () => {
+    hiddenFileInput.current?.click();
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const fileUploaded = event.target.files?.[0];
+    if (fileUploaded) {
+      console.log(fileUploaded);
+      setImageUrl(URL.createObjectURL(fileUploaded));
```

Comment:
how would this work with custom images for different clients? Like if image URL is "Peyman's laptop/grind.jpg", there's no way for me to view it from my computer?

Reply:
fixed